packages/contracts/tasks/deploy/poll/01-constantInitialVoiceCreditProxy.ts

```diff
@@ -40,16 +47,22 @@ deployment
         deployment.getDeployConfigField<number | null>(EContracts.ConstantInitialVoiceCreditProxy, "amount") ??
         DEFAULT_INITIAL_VOICE_CREDITS;
 
-      const constantInitialVoiceCreditProxyContract = await deployment.deployContract(
-        { name: EContracts.ConstantInitialVoiceCreditProxy, signer: deployer },
-        amount.toString(),
-      );
+      const [constantInitialVoiceCreditProxyContract, constantInitialVoiceCreditProxyContractFactory] =
+        await deployConstantInitialVoiceCreditProxy({ amount }, deployer, true);
```

Comment:
I would check if there's a factory already deployed, if so deploy the clone only